sources/web/datalab/templates/nb.html

```diff
@@ -199,37 +180,16 @@
         </div>
         <div id="sidebarArea" style="display:none">
           <div id="sidebarContent">
-            <div class="btn-group">
-              <button id="toggleSidebarButton" type="button" class="toolbar-btn" title="Toggle the size of the sidebar">
-                <i class="material-icons">chevron_left</i>
-              </button>
+            <div>
+              <div class="btn-group pull-right">
+                <button id="hideSidebarButton" type="button" class="toolbar-btn" title="Close sidebar">
```

Comment:
you'll need the `btn btn-default` classes here to style this correctly for both themes, instead of the `toolbar-btn`.
you might also need to adjust the padding for this element, the default button padding is too much for a small `X` button.